pages/en/learn/command-line/run-nodejs-scripts-from-the-command-line.md

```diff
@@ -48,24 +48,12 @@ node -e "console.log(123)"
 
 ## Restart the application automatically
 
-The `node` command has to be re-executed in bash whenever there is a change in the application. To restart the application automatically, use the `nodemon` module.
-
-Install the nodemon module globally to system path:
-
-```bash
-npm i -g nodemon
-```
-
-You can also install nodemon as a development dependency:
+As of nodejs V16, there is a built-in option to automatically restart the application when a file changes. This is useful for development purposes.
+To use this feature, you need to pass the `--watch' flag to nodejs.
 
 ```bash
-npm i --save-dev nodemon
+node --watch app.js
 ```
 
-This local installation of nodemon can be run by calling it from within npm script such as npm start or using npx nodemon.
-
-Run the application using the `nodemon` command followed by the application's file name:
-
-```bash
-nodemon app.js
-```
+So when you change the file, the application will restart automatically.
+If you want to deep dive into this feature, you can read the [official documentation](https://nodejs.org/docs/latest-v20.x/api/cli.html#--watch).
```

Comment:
I noticed that this `--watch` flag is still marked as experimental in the docs, which may be worth mentioning in this learn section. Linking directly to the v20.x documentation means that we're going to need to update this link over time

Reply:
We need to found a good way to allow api link into learn

Reply:
addressed via https://github.com/nodejs/nodejs.org/pull/6132/commits/f90609b15b93b238daa8113a60b1210ecf91597d